portfolio/src/main/webapp/script.js

```diff
@@ -83,22 +30,22 @@ function addGameNameToDOM(gameName)
     document.getElementById('game-container').innerText = gameName;
 }
 
-function displayComments()
+function displayPosts()
 {  
-  fetch('/comments')
+  fetch('/doggo-servlet')
   .then(response => response.json())
-  .then(commentJson => {
-    if(commentJson.length != 0)
+  .then(postJson => {
+    if(postJson.length != 0)
     {
-      populateCommentSection(commentJson)
+      populatePostsSection(postJson);
     }
     else
     {
-      const commentsSection = document.getElementById('comments-section');
-      commentsSection.remove();
-      const commentContainer = document.getElementById('comments-container');
-      commentContainer.innerText = "No comments so far.";
+      const postsSection = document.getElementById('posts-section');
+      postsSection.innerText = "No posts so far.";
     }
+
+    console.log(postJson);
```

Comment:
Usually better to remove these before submitting, though it's useful for debugging :)

Reply:
Done!